### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonathantneal**
- correct typos
- correct capitalization
- add a ’the’ or 'a' where appropriate
- add an oxford comma where appropriate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,24 @@
...
+2. Execute the wizard script. It will ask you a few question and will fill all
    files with your data.
 
     ```sh
    ./postcss-plugin-boilerplate/start
     ```
 
-3. Now it is repository of your plugin with clean Git history.
-   Create GitHub repositpory and push your project there.
+3. Now it is the repository of your plugin with a clean Git history.
```
Comment (ben-eb) [inline README.md]: Your plugin repository will now have a clean Git history.

**Comment by ben-eb** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (iamstarkov) [inline README.md]: btw, why "Git" is with uppercase?

Comment (ben-eb) [inline README.md]: It is spelled like that on the Git website. https://git-scm.com/

Comment (iamstarkov) [inline README.md]: oh, and github help spell it this way too https://help.github.com/articles/https-cloning-errors/

**Pull request merged**